lib/src/base-styles/_nx-colors.scss

```diff
@@ -76,5 +80,9 @@
     //NxBinaryDonutChart variables
     --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-30);
     --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-pink-80);
+
+    // NxGlobalHeader
+    --nx-color-global-header-background: var(--nx-swatch-indigo-20);
```

Comment:
If I'm not mistaken, the header background color is always the same as the tile background color (`--nx-color-component-background`) so I don't think this variable is necessary.

Reply:
> If I'm not mistaken, the header background color is always the same as the tile background color (`--nx-color-component-background`) so I don't think this variable is necessary.

Removed it here: ef41aa01e135d07b51bbcd85a6470c8590920ab8